src/gamma.rs

```diff
@@ -195,6 +198,39 @@ macro_rules! implement { ($kind:ty) => { impl Gamma for $kind {
         let value = unsafe { math::lgamma(self as f64, &mut sign) as Self };
         (value, sign)
     }
+
+    fn trigamma(&self) -> Self {
+
+        use std::f64::consts::PI;
```

Comment:
I think it should be either `f32` or `f64` depending on `$kind`, no?

Reply:
The `$kind` is a type, so it cannot be directly used in the expression `use std::$kind::consts::PI;`.
So added a `consts.rs` module which re-exports PI constant as associated const.